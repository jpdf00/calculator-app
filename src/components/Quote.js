import React from 'react';

const Quote = () => (
  <div className="Quote">
    <p>
      Mathematics is not about numbers, equations, computations, or algorithms:
      <br />
      it is about understanding. â€“ William Paul Thurston
    </p>
  </div>
);

export default Quote;
