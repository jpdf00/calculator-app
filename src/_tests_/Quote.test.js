import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Quote from '../components/Quote';

describe('Quote Component', () => {
  test('It should render', () => {
    const quote = renderer.create(
      <BrowserRouter>
        <Quote />
      </BrowserRouter>,
    ).toJSON();
    expect(quote).toMatchSnapshot();
  });
});
