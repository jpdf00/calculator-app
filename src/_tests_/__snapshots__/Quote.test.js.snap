// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote Component It should render 1`] = `
<div
  className="Quote"
>
  <p>
    Mathematics is not about numbers, equations, computations, or algorithms:
    <br />
    it is about understanding. â€“ William Paul Thurston
  </p>
</div>
`;
